modules/services/hound.nix

```
@@ -0,0 +1,87 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.hound;
+
+  configFile = pkgs.writeText "hound-config.json" (
+    builtins.toJSON {
+      inherit (cfg) max-concurrent-indexers dbpath repos;
+      health-check-url = "/healthz";
+    });
+
+  houndOptions = [
+    "--addr ${cfg.addr}"
+    "--conf ${configFile}"
+  ];
+
+  execStart = pkgs.writeScript "houndd-execstart" ''
+    #!${pkgs.runtimeShell}
+    export PATH="${pkgs.mercurial}/bin:${pkgs.git}/bin''${PATH:+:}$PATH"
+    exec ${cfg.package}/bin/houndd ${concatStringsSep " " houndOptions}
+  '';
+
+in
+
+{
+  meta.maintainers = [ maintainers.adisbladis ];
+
+  options.services.hound = {
+    enable = mkEnableOption "hound";
+
+    package = mkOption {
+      type = types.package;
+      default = pkgs.hound;
+      defaultText = "pkgs.hound";
+      example = literalExample "pkgs.hound";
+      description = "The package to use for the hound binary.";
+    };
+
+    max-concurrent-indexers = mkOption {
+      type = types.int;
+      default = 2;
+      description = "Limit the amount of concurrent houndd indexers";
+    };
+
+    dbpath = mkOption {
```

Best avoid unnecessary abbreviations:
```suggestion
    databasePath = mkOption {
```